x/marker/types/errors.go

```diff
@@ -12,4 +12,5 @@ var (
 	ErrInvalidMarkerStatus     = cerrs.Register(ModuleName, 5, "invalid marker status")
 	ErrAccessTypeNotGranted    = cerrs.Register(ModuleName, 6, "access type not granted")
 	ErrMarkerNotFound          = cerrs.Register(ModuleName, 7, "marker not found")
+	ErrDuplicateEntry          = cerrs.Register(ModuleName, 8, "duplicate entry")
```

Comment:
duplicate entry in allowed list ..maybe .for more precision

Reply:
Was thinking of keeping it more generic for possible usages else where...

Reply:
I ended up using the Wrap on the error with more detail at the point it it hit: `ErrDuplicateEntry.Wrap("all allow list addresses must be unique")`